src/_playground/Routes.js

```diff
@@ -356,6 +356,11 @@ export class Routes extends Component {
                                 onClick={this.toggleNavVis}
                                 option='light'
                                 type='standard' />
+                            <a
+                                className='frDocs-Menu__screen-reader-only-focusable'
+                                href='#frDocs-Content'>
+                                Skip Navigation Links
+                            </a>
```

Comment:
I think this block should come before the toggle `Button` in case you're on a small screen.